src/renderers/shaders/ShaderChunk/lights_physical_pars_fragment.glsl

```diff
@@ -0,0 +1,42 @@
+struct PhysicalMaterial {
+	vec3	diffuseColor;
+	float	specularRoughness;
+	vec3	specularColor;
+	float	clearCoatWeight;
+	float	clearCoatRoughness;
+};
```

Comment:
`specularRoughness` ?
`clearCoatRoughness` ?
`clearCoatWeight` ?

Modeling change. :/

We previously discussed creating a separate model for more sophisticated users, and keeping this as a 4-parameter model.

Again, I do not care if the model is changed. This is not the place to do it.